src/fides/api/util/aws_util.py

```diff
@@ -95,10 +97,16 @@ def get_s3_client(
     If an `assume_role_arn` is provided, the secrets will be used to
     assume that role and return a Session instantiated with that role.
     """
+
```

Comment:
i would callout in the docstring here that we fallback/default to the configured storage credentials.

i don't think we've got any cases yet where we're calling `get_s3_client` outside of the "storage" functionality, but it's not inconceivable that we'd want e.g. an s3 _monitor_ to call this utility function, and it may not rely on these same credentials.

i don't think that hypothetical should prevent us from making this update, but i do think that we should make it clear to any potential future callers of this very broad utility function that they need to be a bit careful about the assumptions it makes on where it's going to look for the `assume_role_arn`...